### PR TITLE
Fixed support aircraft being huntable

### PR DESCRIPTION
**Pull request by Mailaender**
This is a followup to https://github.com/OpenRA/OpenRA/pull/11800/.

### CODE REVIEW

File: mods/ra/rules/defaults.yaml [hunk 1]
```diff
@@ -541,7 +541,6 @@
 	Inherits@bounty: ^GlobalBounty
 	Inherits@selection: ^SelectableCombatUnit
 	Inherits@handicaps: ^PlayerHandicaps
-	Huntable:
```
Comment (abcdefg30) [inline mods/ra/rules/defaults.yaml:544]: We probably need to add this to `^Plane` then?

Comment (Mailaender) [reply mods/ra/rules/defaults.yaml:544]: Done.

**Review by abcdefg30 — APPROVED** [review]
Changes lgtm.

**Review by Smittytron — APPROVED** [review]

**Comment by Smittytron** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/389951b90b38221357a1290bb9e16dadd707f07c

**Pull request merged**